### PR TITLE
Add hector settings

### PR DESCRIPTION
**Pull request by yding37**
Enable setting the following values for hector:
  - cassandraThriftSocketTimeout
  - useSocketKeepalive
Enabled HectorLockManager

### CODE REVIEW

File: stack/config/src/main/resources/usergrid-default.properties [hunk 1]
```diff
@@ -62,6 +62,15 @@ cassandra.mutation.flushsize=2000
 #Note that if this is deployed in a production cluster, the RF on the keyspace MUST be updated to use an odd number for it's replication Factor.
 #Even numbers for RF can potentially case the locks to fail, via "split brain" when read at QUORUM on lock verification
 cassandra.lock.keyspace=Locks
+#locking read & write policies
+cassandra.lock.readcl=QUORUM
```
Comment (snoopdave) [inline stack/config/src/main/resources/usergrid-default.properties]: both of these should be local quorum by default

**Comment by snoopdave** [comment]
what version of Hector adds the new cassandraThriftSocketTimeout parameters?

**Comment by yding37** [comment]
I'm not too sure actually, although the version that's shipped with
usergrid has that setting.

On Thu, Apr 16, 2015 at 12:42 PM, David M. Johnson <notifications@github.com

**Comment by yding37** [comment]
is this necessary for build purposes?

**Comment by GERey** [comment]
Hello! If you could update this PR to address the conflicts I would be more than happy to accept the PR. Thanks!

**Pull request merged**